common/warp_utils/Cargo.toml

```diff
@@ -18,3 +18,4 @@ tokio = { version = "1.14.0", features = ["sync"] }
 headers = "0.3.2"
 lighthouse_metrics = { path = "../lighthouse_metrics" }
 lazy_static = "1.4.0"
+serde_array_query = { git = "https://github.com/sigp/serde_array_query", rev = "67c3eaff7dc42d256509adad2e40a70e973248ec" }
```

Comment:
I think we should publish `serde_array_query` to crates.io before we merge. Would you mind adding CI for `serde_array_query`, https://github.com/sigp/serde_array_query/issues/6? :pray: